lib/generators/data_script/templates/data_script_spec.erb

```diff
@@ -0,0 +1,15 @@
+require "rails_helper"
+
+describe <%= class_name %>Script do
+  it "runs" do
+    described_class.call
+  end
+
+  it "changes nothing in dry run mode" do
```

Comment:
This part is what made the generator seem more valuable to me. It's an interesting way of enforcing dry_run functionality.